docs/user-guide/misc/comparison.md

```diff
@@ -0,0 +1,33 @@
+# Comparison with other tools
+
+These are several libraries and tools that share similar functionalities with Polars. This often leads to questions from data experts about what the differences are. Below is a short comparison between some of the more popular data processing tools and Polars, to help data experts make a deliberate decision on which tool to use.
+
+You can find performance benchmarks (h2oai benchmark) of these tools here: [Polars blog post](https://pola.rs/posts/benchmarks/) or a more recent benchmark [done by DuckDB](https://duckdblabs.github.io/db-benchmark/)
+
+### Pandas
+
+Pandas stands as a widely-adopted and comprehensive tool in Python data analysis, renowned for its rich feature set and strong community support. However, due to its single threaded nature, it can struggle with performance and memory usage on medium and large datasets.
+
+Polars, in contrast, is tailored for high-performance multithreaded computing on single nodes, offering significant speed and memory efficiency improvements, especially for medium to large data operations.
```

Comment:
I would also mention our API as being more composable and more strict. Leading to more expressiveness and lesser (schema related) bugs.